package test;

import java.sql.SQLException;

import javax.naming.Context;
import javax.naming.InitialContext;
import javax.naming.NamingException;
import javax.sql.DataSource;

public class test {
	
	public static void main(String[] args) {
		
		
		try {
			
		} catch (Exception e) {
			// TODO: handle exception
		}
	}
	 
	
}
